quarkus/runtime/src/main/resources/keycloak-default-client-policies.json

```diff
@@ -0,0 +1,18 @@
+{
+  "policies": [
```

Comment:
It will be good to avoid a need to have the builtin JSON files for policies and profiles on multiple places. I hope there is no need to have the files inside `quarkus/runtime` module, but is sufficient to keep in keycloak-services? The duplication adds the potential for various bugs and inconsistencies IMO...

Reply:
@mposolda I agree with you. In Quarkus distribution, I will access `keycloak-services.jar` and read `keycloak-default-client-policies.json` and `keycloak-default-client-profiles.json` enclosed in this jar.

However, I am still novice in Quarkus so that I do not know how to access resources in `keycloak-services.jar` on Quarkus distribution. I will try to find out it, but I would be happy if you would tell it.

Reply:
Thanks for looking at this! Please let me know If you don't manage to figure this, I will try to help you more actively for this.

Just as a hint, I would start with build quarkus keycloak distribution and debug it manually if it is possible to find the classes or not. It can be done by:
```
mvn clean install -DskipTests=true -Pdistribution,quarkus
```
Then in the `distribution/server-x-dist/target/keycloak.x-13.0.0-SNAPSHOT.zip` you have the built server. When you unpack it and run with the `./kc.sh --debug` you can attach remote debugger to it. Hope it helps a bit...